plugins/meteor_engage_warn.py

```diff
@@ -68,7 +68,7 @@ def _display_meteor_pos_warning_msg(self, end_pos: Dict[str, float]) -> bool:
         return: True if the user accepts, False if the move should be cancelled.
         """
         pos_str = []
-        for axis in ("x", "y", "z", "rx", "ry", "rz"):
+        for axis in ("x", "y", "m", "rx", "ry", "rm"):
```

Comment:
`for axis in ("x", "y", "m", "rx", "ry", "rm"):`
Take into account that the driver>zeiss.py works with axes definition of "z" instead of "m". I am not sure if you run into issues regarding that.

Reply:
We need both z and m axes for the movement. I checked in the config, all the relevant axes are defined.  It is an interesting point.